content/work/timedesigner.md

```diff
@@ -7,13 +7,49 @@ date: 2018-07-15
 image:
   src: projects/tegy/grid-desktop.jpg
   alt: School schedule grid
+fullgrid:
+  - img: work/timedesigner/full-grid.jpg
+    alt: a completed scenario with experiences laid out in a grid
+experiencegrid:
+  - img: work/timedesigner/experience-editing.jpg
+    alt: browser inspector showing grid markup and css for a single experience
+emptygridoverlay:
+  - img: work/timedesigner/empty-grid-overlay.jpg
+    alt: Firefox Grid Inspector tool including an overlay of the grid lines
+scenariogrid:
+  - img: work/timedesigner/setting-up-scenario-grid.jpg
+    alt: an empty resource planning grid showing the start and end times
+overlapexperience:
+  - img: work/timedesigner/overlap-experience.jpg
+    alt: a row with overlapping experiences
 people:
   - &furman
     name: Furman Brown
-    # face: furman-brown.jpg
+    face: furman-brown.jpg
     url: https://www.tegy.tools/profile/furman/profile
     title: Founder
     venue: *client
+tags:
+  - _post
+  - Case Study
+sample:
+  desktop:
+    src: tegy/timedesigner-desktop.jpg
```

Comment:
Is it intentional for this screenshot to have a span hovered/inspected? It looks out of place to me, and I can't figure out what it adds.

Reply:
I photoshopped it out of the mobile view (although we might not use that now) so I agree. @SondraE do you have this without the inspector tooltip?